__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const db = require("../db/connection");
+
+afterAll(() => {
+  db.end();
```

Comment:
Remember as we have opening this callback up with curly brackets it is no longer implicitly returning db.end() - we need to explicitly tell it to!

Reply:
Great, thanks! Just read the Supertest notes again. Lost the brackets and changed to afterAll(() => db.end());